bin/syclcc-clang

```diff
@@ -697,6 +697,8 @@ class syclcc_config:
   def hipsycl_plugin_path(self):
     if sys.platform.startswith('win32'):
       return os.path.join(self.hipsycl_installation_path, "bin", "hipSYCL_clang.dll")
+    elif sys.platform == "darwin":
+      return os.path.join(self.hipsycl_installation_path, "lib", "libhipSYCL_clang.dylib")
```

Comment:
We already have this fix in current develop. I can merge that into `generic-llvm-sscp` branch if you like.

Reply:
Yes, please do.

Reply:
Done :)